code/drasil-database/lib/Database/Drasil/ChunkDB.hs

```diff
@@ -213,6 +213,8 @@ cdb s t c u d ins gd tm ci lc r =
     _refbyTable = Map.empty,
     _refTable = idMap r
   }
+  where
+    termsHACK = map nw d
```

Comment:
2. When we change the `UID`s of the existing data definitions, we see runtime errors asking for missing `UID`s. That's caused by missing `IdeaDict`s. Here, we are creating them.